tokens/color/palette.json

```diff
@@ -0,0 +1,443 @@
+          "$value": "rgba(14, 132, 32, 1)"
+        },
+        "9": {
+          "themeable": false,
+          "$value": "rgba(0, 128, 19, 1)"
+        },
+        "10": {
+          "themeable": false,
+          "$value": "rgba(12, 109, 26, 1)"
+        },
+        "11": {
+          "themeable": false,
+          "$value": "rgba(0, 103, 15, 1)"
+        },
+        "12": {
+          "themeable": false,
+          "$value": "rgba(24, 96, 35, 1)"
+        },
+        "13": {
+          "themeable": false,
+          "$value": "rgba(10, 95, 23, 1)"
+        },
+        "14": {
+          "themeable": false,
+          "$value": "rgba(27, 86, 36, 1)"
+        },
+        "15": {
+          "themeable": false,
+          "$value": "rgba(0, 87, 13, 1)"
+        },
+        "16": {
+          "themeable": false,
+          "$value": "rgba(32, 68, 38, 1)"
+        }
+      },
+      "teal": {
+        "1": {
+          "themeable": false,
+          "$value": "rgba(15, 149, 161, 1)"
+        }
+      },
+      "blue": {
+        "1": {
+          "themeable": false,
+          "$value": "rgba(229, 239, 249, 1)"
+        },
+        "2": {
+          "themeable": false,
+          "$value": "rgba(217, 232, 247, 1)"
+        },
+        "3": {
+          "themeable": false,
+          "$value": "rgba(219, 229, 239, 1)"
+        },
+        "4": {
+          "themeable": false,
+          "$value": "rgba(209, 227, 245, 1)"
+        },
+        "5": {
+          "themeable": false,
+          "$value": "rgba(207, 221, 236, 1)"
+        },
+        "6": {
+          "themeable": false,
+          "$value": "rgba(199, 217, 235, 1)"
+        },
+        "7": {
+          "themeable": false,
+          "$value": "rgba(153, 204, 255, 1)"
+        },
+        "8": {
+          "themeable": false,
+          "$value": "rgba(94, 166, 237, 1)"
+        },
+        "9": {
+          "themeable": false,
+          "$value": "rgba(46, 150, 255, 1)"
+        },
+        "10": {
+          "themeable": false,
+          "$value": "rgba(102, 153, 204, 1)"
+        },
+        "11": {
+          "themeable": false,
+          "$value": "rgba(0, 102, 204, 1)"
+        },
+        "12": {
+          "themeable": false,
+          "$value": "rgba(36, 89, 143, 1)"
+        },
+        "13": {
+          "themeable": false,
+          "$value": "rgba(24, 74, 116, 1)"
+        },
+        "14": {
+          "themeable": false,
+          "$value": "rgba(27, 68, 103, 1)"
+        },
+        "15": {
+          "themeable": false,
+          "$value": "rgba(30, 58, 81, 1)"
+        }
+      },
+      "purple": {
+        "1": {
+          "themeable": false,
+          "$value": "rgba(166, 121, 210, 1)"
+        },
+        "2": {
+          "themeable": false,
+          "$value": "rgba(125, 66, 184, 1)"
+        }
+      },
+      "gray": {
+        "white": {
+          "themeable": false,
+          "$value": "rgba(255, 255, 255, 1)"
+        },
```

Comment:
Should "white" be nested inside "gray"? Wouldn't this generate something like `color-palette-gray-white`?